moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -236,6 +421,26 @@ ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcp
                         "greater than zero. Check yaml file.");
     return nullptr;
   }
+  return parameters;
+}
+
+ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcpp::Node::SharedPtr& node,
+                                                                     const rclcpp::Logger& logger,
+                                                                     std::string ns, /* = "moveit_servo"*/
+                                                                     bool dynamic_parameters /* = true */)
+{
+  auto parameters = std::shared_ptr<ServoParameters>(new ServoParameters(logger, ns));
+  auto node_parameters = node->get_node_parameters_interface();
+  // Get the parameters (organized same order as YAML file)
+  try
+  {
+    declare(ns, node_parameters, parameters);
+  }
+  catch (rclcpp::exceptions::ParameterAlreadyDeclaredException& e)
+  {
+    RCLCPP_WARN_STREAM(logger, "Parameters already declared.");
+  }
```

Comment:
When running the joystick servo example, this will throw an exception that suggests the paramaters have already been declared before the declare function gets to do it. Catching this right now just as a test. Will try to fix later.